fate/src/main/java/org/apache/accumulo/fate/AdminUtil.java

```diff
@@ -110,25 +110,36 @@ public String getTop() {
   public static class FateStatus {
 
     private final List<TransactionStatus> transactions;
-    private final Map<Long,List<String>> danglingHeldLocks;
-    private final Map<Long,List<String>> danglingWaitingLocks;
+    private final Map<String,List<String>> danglingHeldLocks;
+    private final Map<String,List<String>> danglingWaitingLocks;
+
+    private static Map<String,List<String>> convert(Map<Long,List<String>> danglocks) {
+      if (danglocks.isEmpty()) {
+        return Collections.emptyMap();
+      }
+
+      Map<String,List<String>> ret = new HashMap<>();
+      for (Entry<Long,List<String>> entry : danglocks.entrySet()) {
+        ret.put(String.format("%016x", entry.getKey()), Collections.unmodifiableList(entry.getValue()));
```

Comment:
I added javadocs.  The specific reason I made the change was so that when I saw an assertion failure message in a test I could easily find the transaction id in the logs.  Before this change I had to manually convert base 10 to base 16 to find the corresponding info in the logs.